packages/common/src/types/command/PartialTargetDescriptor.types.ts

```diff
@@ -371,3 +381,25 @@ export type PartialTargetDescriptor =
   | PartialRangeTargetDescriptor
   | PartialListTargetDescriptor
   | ImplicitTargetDescriptor;
+
+export interface PartialPrimitiveDestinationDescriptor {
+  type: "primitiveDestination";
```

Comment:
I guess we could do that. I was just thinking that when you have a serialized log it might be a bit more obvious if we have different names between destinations and targets

Reply:
update: let's do it